examples/pipelines/fl_cross_silo_basic/config.yaml

```diff
@@ -13,13 +13,22 @@ federated_learning:
     datastore: "workspaceblobstore"
 
   silos:
-    - compute: cpu-cluster-silo-1
-      datastore: workspaceblobstore
-      training_data_path: "azureml://datastores/workspaceblobstore/paths/azureml/ee95ed94-03d5-4b85-ad5e-883880b44735/extracted_data/"
-    - compute: cpu-cluster-silo-2
-      datastore: workspaceblobstore
-      training_data_path: "azureml://datastores/workspaceblobstore/paths/azureml/ee95ed94-03d5-4b85-ad5e-883880b44735/extracted_data/"
+    - compute: silo-wus2
+      datastore: workspaceblobstoresilo1
```

Comment:
Sorry, I forgot to mention that part yesterday. Since the auto-provisioning script does not create datastores yet, let's keep using the default `workspaceblobstore` for this and other silos.

Reply:
Sure.